addons/mass_mailing/models/mailing.py

```diff
@@ -677,6 +677,9 @@ def _process_mass_mailing_queue(self):
             if len(mass_mailing._get_remaining_recipients()) > 0:
                 mass_mailing.state = 'sending'
                 mass_mailing.action_send_mail()
+                # Re-trigger the cron to process remaining recipients
+                cron = self.env.ref('mass_mailing.ir_cron_mass_mailing_queue')
+                cron._trigger(fields.Datetime.now() + relativedelta(hours=1))
```

Comment:
Can we put this as something configurable? Previously, we adjusted the cron for this. Now, as this is decoupled, it doesn't serve, so maybe 1 hour is not the best suitable one, depending on the mail server limits. I would also disable the cron at all if we make this.

Reply:
I'm thinking of a case where you send the mail at 23:30 for 500 mails, and the cron launches at 23:45 (after 24 hours since last launch), sending another 500 mails, making a total of 1000 in an hour instead of 500. Or the next date is updated also when there are triggers?

Reply:
The next date is not, so indeed the cron would run 3 times, 2 by triggers and 1 by the normal cron interval